extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -636,6 +648,10 @@ public void onFetchPage(@Nonnull Downloader downloader) throws IOException, Extr
         if (subtitlesInfos.isEmpty()) {
             subtitlesInfos.addAll(getAvailableSubtitlesInfo());
         }
+
+        if (!this.getDashMpdUrl().isEmpty()) {
+            dashResult = DashMpdParser.getStreams(this.getDashMpdUrl());
```

Comment:
Do not calculate `getDashMpdUrl()` twice

Reply:
Yeah, I'll fix that later (today?) and add Mauricio as co-author of 8a9876067df764a81ed2430f39df267bd27bd288 as that part was mostly from #358.

Reply:
Have you been able to make the fix yet @wb9688 ?